client/lib/credit-card-details/validation.js

```diff
@@ -8,49 +8,91 @@ import creditcards from 'creditcards';
 import { capitalize, compact, inRange, isArray, isEmpty } from 'lodash';
 import i18n from 'i18n-calypso';
 
-function creditCardFieldRules() {
+/**
+ * Internal dependencies
+ */
+import { isEbanx, isValidCPF } from 'lib/credit-card-details/ebanx';
+
+function ebanxFieldRules() {
 	return {
-		name: {
-			description: i18n.translate( 'Name on Card', {
-				context: 'Upgrades: Card holder name label on credit card form',
-				textOnly: true,
-			} ),
-			rules: [ 'required' ],
+		document: {
+			description: i18n.translate( 'Taxpayer Identification Number' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **5** times:
`translate( 'Taxpayer Identification Number (TIN)' )` *ES Score: 28*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).